ncloud/resource_ncloud_subnet.go

```diff
@@ -0,0 +1,271 @@
+package ncloud
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/NaverCloudPlatform/ncloud-sdk-go-v2/ncloud"
+	"github.com/NaverCloudPlatform/ncloud-sdk-go-v2/services/vpc"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceNcloudSubnet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNcloudSubnetCreate,
+		Read:   resourceNcloudSubnetRead,
+		Update: resourceNcloudSubnetUpdate,
+		Delete: resourceNcloudSubnetDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		CustomizeDiff: resourceNcloudSubnetCustomizeDiff,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
```

Comment:
![image](https://user-images.githubusercontent.com/34295380/88611173-6fbbcc00-d0c3-11ea-8dd4-764a2e852c08.png)
```suggestion
				Optional:     true,
```